content/doc/developer/plugin-governance/managing-permissions.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: developersection
+title: Managing plugin permissions
+references:
+- url: https://github.com/jenkins-infra/repository-permissions-updater
+  title: Repository Permissions Updater
+- url: ../adopt-a-plugin
+  title: Adopting a plugin
+---
+
+This page documents managing permissions for plugin maintainers.
+There are 2 main permissions available to plugin contributors:
+
+* GitHub permissions. 
```

Comment:
Make it clear that this is a definition term followed by the definition of the term.  I wish that the layout of ASCIIDoc definition lists ("labeled lists") were better, but this is technique provides an easier to read layout that adoc labeled lists (at least for me)
```suggestion
* **GitHub permissions**. 
```